rpcs3/rpcs3qt/welcome_dialog.ui

```diff
@@ -242,6 +242,16 @@
          </property>
         </widget>
        </item>
+       <item alignment="Qt::AlignLeft">
```

Comment:
Why not use the dialog button box?

Reply:
I simply cloned and renamed the existing `okay` button. What is your idea?